web/src/games/kriegspiel/instructions.md

```diff
@@ -0,0 +1,55 @@
+# Le Jeu de la Guerre
+
+Try it on the [GitHub page](https://iamcxds.github.io/kriegspiel).
+
+A WEB implementation of Guy Debord's A Game of War ([Le Jeu de la Guerre](https://www.wikiwand.com/fr/Le_Jeu_de_la_guerre_(livre))). Powered by [boardgame.io](https://boardgame.io/). The graphic is based on svg+emoji, with [Pico8's Palette](https://pico-8.fandom.com/wiki/Palette).
```

Comment:
This seems a bit out of place for users going to FBG? They dont really care we use boardgame.io and etc, I think you can delete this first section...

Reply:
And by the way, how to insert images in instructions.md?